### ISSUE TITLE
[JST-23] Validator getTableById does not map chainId nor tableId

### ISSUE THREAD

**Issue by dtbuchholz** [tablelandnetwork/js-tableland#470]
## Summary

The Validator's `getTableById` method does not work and maps `undefined` to both `chainId` and `tableId`, making the method unusable.

## Details

Set up a quick `Database` and create a `Validator`with hardcoded values:

```
const { Database, Validator, helpers } = require("@tableland/sdk");

const db = new Database({
  baseUrl: helpers.getBaseUrl(1),
});
const validator = new Validator(db.config);
const name = await validator.getTableById(1, 1);
```

When the query is made, it will error out with a message of `no chain id in path`—and you can see the `tableId` is also `undefined`(`https://tableland.network/api/v1/tables/undefined/undefined`):

```
ApiError: no chain id in path
    at fun (/Users/dtb/Desktop/hardhat-fevm/node_modules/@tableland/sdk/src/validator/client/fetcher.ts:214:15)
    at processTicksAndRejections (node:internal/process/task_queues:95:5)
    at getTable (/Users/dtb/Desktop/hardhat-fevm/node_modules/@tableland/sdk/src/validator/tables.ts:63:20)
    at Validator.getTableById (/Users/dtb/Desktop/hardhat-fevm/node_modules/@tableland/sdk/src/validator/index.ts:84:12)
    at main (/Users/dtb/Desktop/hardhat-fevm/scripts/read.js:14:19) {
  headers: HeadersList {
    [Symbol(headers map)]: Map(14) {
      'date' => 'Thu, 27 Apr 2023 16:26:05 GMT',
      'content-type' => 'text/plain; charset=utf-8',
      'content-length' => '34',
      'connection' => 'keep-alive',
      'access-control-allow-headers' => 'Accept, Accept-Language, Content-Type, Authorization',
      'access-control-allow-methods' => 'GET, POST, OPTIONS',
      'access-control-allow-origin' => '*',
      'trace-id' => '5020d094-5b76-46af-b426-1f169fd77766',
      'cf-cache-status' => 'DYNAMIC',
      'report-to' => '{"endpoints":[{"url":"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=jbrRodgQGBOwcuM9Peh%2FNXLsH%2BOD2qh8ruvBltu2%2F1ovAUQ9uGpS5MG%2BrJmVMSN6zZAVdJ883uJc2wx2nCYasWSv04TiRP1HmnSalSZpyxvPWnh12MGHj3XVXH8%2BwgWiYcOXLQ%3D%3D"}],"group":"cf-nel","max_age":604800}',
      'nel' => '{"success_fraction":0,"report_to":"cf-nel","max_age":604800}',
      'server' => 'cloudflare',
      'cf-ray' => '7be86cd6ca189e52-SJC',
      'alt-svc' => 'h3=":443"; ma=86400, h3-29=":443"; ma=86400'
    },
    [Symbol(headers map sorted)]: null
  },
  url: 'https://tableland.network/api/v1/tables/undefined/undefined',
  status: 400,
  statusText: 'Bad Request',
  data: { message: 'no chain id in path' }
}
```

## Environment

* Node 18
* @tableland/sdk ^4.2.0
* Hardhat ^2.13.0

<sub>From [SyncLinear.com](https://synclinear.com) | [JST-23](https://linear.app/tableland/issue/JST-23/validator-gettablebyid-does-not-map-chainid-nor-tableid)</sub>

**Comment by joewagner** [comment]
@dtbuchholz Looks like you are not passing in valid arguments.
The usage is:
```
const response = await api.getTableById({
  chainId: 1,
  tableId: "1",
});
```

Should we explicitly check and throw an Error with a different message?

**Comment by joewagner** [comment]
Fixed by #472
